source/cdaudio.h

```diff
@@ -19,7 +19,11 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 */
 
 int CDAudio_Init(void);
+#ifndef __WII__
 void CDAudio_Play(byte track, qboolean looping);
+#else
+void CDAudio_Play(char* filename, qboolean looping)
```

Comment:
Don't do this — add CD_PlayFromString from PSP instead of changing the spec for CDAudio_Play